src/popup/pages/AccountDetailsMultisigTransactions.vue

```diff
@@ -133,6 +155,11 @@ export default defineComponent({
     });
 
     onMounted(() => {
+      if (innerScrollElem.value && appInnerElem.value) {
+        appInnerElem.value.addEventListener('scroll', () => {
```

Comment:
Please add debounce to scroll watcher.

Reply:
I think this is better achieved using [throttle](https://lodash.com/docs#throttle) instead of debounce

Reply:
Yes, this was a mental shortcut :)